core/src/main/java/de/danoeh/antennapod/core/service/playback/LocalPSMP.java

```diff
@@ -919,12 +919,14 @@ private boolean genericInfoListener(int what) {
 
     private final MediaPlayer.OnErrorListener audioErrorListener =
             (mp, what, extra) -> {
-                if(mp.canFallback()) {
-                    mp.fallback();
-                    return true;
-                } else {
-                    return genericOnError(mp, what, extra);
+                if (mp.canFallback()) {
+                    try {
+                        mp.fallback();
+                        return true;
+                    } catch (Exception ignore) {
+                    }
```

Comment:
please add a comment why it is ignored or what the `try-catch` is for.
but isn't it rather odd that importing the DB leads to a crash here? shouldn't this maybe prevented way further up the chain? why is there even a mediaplayer when you do this?